setupbase.py

```diff
@@ -375,6 +375,15 @@ def npm_components(self):
                 shutil.rmtree(bower_pkg)
             shutil.copytree(npm_pkg, bower_pkg)
 
+    def patch_codemirror(self):
+        """Patch CodeMirror until https://github.com/codemirror/CodeMirror/issues/4454 is resolved"""
+        
+        try:
+            run(['cp', 'scripts/codemirror.js', 'notebook/static/components/codemirror/lib'], cwd=repo_root)
```

Comment:
Also `tools` is perhaps a more appropriate place than 'scripts'. 'scripts' is for entrypoints that we install (like `jupyter-notebook` etc.).